src/main/java/org/broadinstitute/hellbender/utils/pairhmm/VectorLoglessPairHMM.java

```diff
@@ -101,12 +101,16 @@ public VectorLoglessPairHMM() throws UserException.HardwareFeatureException {
                     logger.info("libVectorLoglessPairHMM found in JVM library path");
                 } catch (UnsatisfiedLinkError e) {
```

Comment:
Tricky to mock a static method.  See our conversation about how I wish java had a concept for "abstract static method".